### ISSUE TITLE
Irrelevant text while sharing a location- "I am here now"

### ISSUE THREAD

**Issue by humble-GOAT** [organicmaps/organicmaps#9515]
⚠ Have you searched for similar, already existing issues?

**Is your feature request related to a problem? Please describe.

When sharing a location with others, the current message displayed is "I am here now." This can be misleading, especially when the location being shared is not where the user is currently located. I often share locations where I am not present at the moment  and the message could cause confusion.

**Describe the ideal solution**
A clear and concise description of what you want to see in Organic Maps.
Ideal solution would be a link with location.

**Additional context**

![screen](https://github.com/user-attachments/assets/5a795fda-b56c-462f-8032-0f5f8fe49406)

**Comment by meenbeese** [comment]
I agree that this can be misleading. How about a message such as "Check out my location..." instead? I can submit a PR if it works.

**Comment by raar1** [comment]
I think "Check out my location" still implies that you are at that location. Honestly I would rather OM did not add a message at all - just the link, and possibly the name of the POI or something.

**Comment by Misalf-git** [comment]
> - just the link, and possibly the name of the POI or something.

👍︎

**Comment by pastk** [comment]
There is a PR that fixes it but it has been stalled for a long time and is looking for someone to revive it and finish
- #1821

**Comment by ChildishGiant** [comment]
This is one of my major gripes with OM (which I guess shows how much I like this app). I am almost never at a location I share so this message normally just saps a bit of my time while I delete it
